### ISSUE TITLE
Fails on existential conditional in throw calls

### ISSUE THREAD

**Issue by zackbloom** [decaffeinate/decaffeinate#1096]
Decaffeinate is crashing on my CoffeeScript input:

```coffee
throw(res.data?.error)
```

[(repl)](http://decaffeinate-project.org/repl/#?evaluate=true&stage=full&code=throw(res.data%3F.error))

I get this error:

```
AddVariableDeclarationsStage failed to parse: Unexpected token (1:6)
```

**Comment by alangpierce** [comment]
Thanks for reporting! Should be an easy fix I think.

**Comment by alangpierce** [comment]
Will be fixed when #1097 merges.
